assets.go

```diff
@@ -175,6 +175,16 @@ func LoadShader(vertSrc, fragSrc string) *gl.Program {
 	vertShader := Gl.CreateShader(Gl.VERTEX_SHADER)
 	Gl.ShaderSource(vertShader, vertSrc)
 	Gl.CompileShader(vertShader)
+	var success int32
+	Gl.GetShaderiv(vertShader, Gl.COMPILE_STATUS, &success)
+	if success != int32(Gl.TRUE) {
+		var maxLength int32
+		Gl.GetShaderiv(vertShader, Gl.INFO_LOG_LENGTH, &maxLength)
+		log.Println("Error during vertex shader compilation")
+		errorLog := make([]byte, maxLength)
+		Gl.GetShaderInfoLog(vertShader, maxLength, &maxLength, (*uint8)(Gl.Ptr(errorLog)))
+		log.Print(string(errorLog))
```

Comment:
The `log` package automatically inserts newline characters.

From [log.Output](https://golang.org/pkg/log/#Output)
> A newline is appended if the last character of s is not already a newline.